app/assets/javascripts/cascade/level/career_blocks.js

```diff
@@ -0,0 +1,202 @@
+$(function () {
+  if ($(".career-block-widget__container").children().length > 1) {
+    $(".career-block-widget__container").addClass(
+      "career-block-widget__container--3-or-more-items"
+    );
+  }
+
+  $.each($(".career-block-widget"), function (ind) {
+    $(this).attr("id", "career-block-widget__" + parseInt(ind + 1));
+    var id = $(this).attr("id");
+    // console.log("current id: " + id);
+    // console.log("this " + id);
+    var careerCode = $(this)
+      .find(".career-block-widget__career-code")
+      .text()
+      .replace(/\s/g, "");
+    console.log("careerCode " + careerCode);
+    var URL =
+      "https://services.onetcenter.org/ws/online/occupations/" +
+      careerCode +
+      "?client=chapman";
+    // console.log("url: " + URL);
+
+    var bindedOnSuccess = onSuccess.bind(this);
+    $.ajax({
+      type: "GET",
+      url: URL,
+      data: {
+        get_param: "value"
+      },
+      async: true,
+      dataType: "json",
+      timeout: 10000,
+      error: function (jqxhr, statustext, err) {
+        console.log(id + " error");
+        if (jqxhr.status === 422) {
+          console.log(JSON.parse(jqxhr.responseText));
+        } else if (jqxhr.status) {
+          console.log({
+            error: "Call to " + url + " failed with error code " + jqxhr.status
+          });
+        } else if (statustext === "timeout") {
+          console.log({
+            error: "Call to " + url + " failed with no response from server"
+          });
+        } else if (err) {
+          console.log({
+            error: "Call to " + url + " failed with reason: " + err
+          });
+        } else {
+          console.log({
+            error: "Call to " + url + " failed with unknown reason"
+          });
+        }
+      },
+      success: bindedOnSuccess
+    });
+
+    /// ON SUCCESS CALLBACK
+    function onSuccess(data) {
+      $(this)

```

Comment:
You should put this in a `var` since it's faster than selecting the `$(this)` multipletimes 
```javascript
  var $this = $(this)
  $this.find('.career-block-widget__text')
```

Reply:
Good to know, thanks @cbryant24. Followed up with the commit below